fastlane/lib/fastlane/actions/appetize_url_generator.rb

```diff
@@ -0,0 +1,107 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class AppetizeUrlGeneratorAction < Action
+      def self.run(params)
+        link = "https://appetize.io/embed/#{params[:public_key]}"
+
+        url_params = []
+        url_params << "autoplay=true"
+        url_params << "orientation=#{params[:orientation]}"
+        url_params << "device=#{params[:device]}"
+        url_params << "deviceColor=#{params[:color]}"
+        url_params << "scale=#{params[:scale]}"
+        url_params << "launchUrl=#{params[:launch_url]}" if params[:launch_url]
+        url_params << "language=#{params[:language]}" if params[:language]
```

Comment:
Seems like they should be escaped.  e.g. if the `launchUrl` contained a `?param1=foo&param2=bar` I don't see how the downstream could handle that properly.